openAPI/quran.com/v4.json

```diff
@@ -2298,26 +2298,26 @@
         "tags": [
           "Quran"
         ],
-        "description": "Get a single #model:D7oy5DF6aGMhfLwDB of all ayah.\n\nSee #endpoint:5YnxJJGPMCLnM6PNE\nendpoint to fetch available tafsirs.\n\nYou can also include more fields of tafsir using `fields` query string.",
+        "description": "Get a single tafsir\n\nSee https://api-docs.quran.com/docs/quran.com_versioned/tafsirs to fetch available tafsirs.",
```

Comment:
I think Tafsir should be capitalized across the entire doc.

Reply:
Semi-complete, I will triple-check tomorrow